fe1-web/model/objects/Chirp.ts

```diff
@@ -0,0 +1,87 @@
+import { Timestamp } from './Timestamp';
+import { Hash } from './Hash';
+
+/**
+ * Object to represent a Chirp.
+ */
+
+export interface ChirpState {
+  id: string;
+  sender: string;
+  text: string;
+  time: number;
+  likes: number;
+  parentId: string;
+}
+
+export class Chirp {
+  public readonly id: Hash;
+
+  // The sender's public key
+  public readonly sender: string;
```

Comment:
if it's a public key, why not use the `PublicKey` object for clarity (and consistency) ?

Reply:
That's true.